pkg/internal/controller/metrics/metrics.go

```diff
@@ -89,6 +91,13 @@ func init() {
 		// expose process metrics like CPU, Memory, file descriptor usage etc.
 		collectors.NewProcessCollector(collectors.ProcessCollectorOpts{}),
 		// expose Go runtime metrics like GC stats, memory stats etc.
-		collectors.NewGoCollector(),
+		collectors.NewGoCollector(
```

Comment:
I don't think we want this change - Any config we put in here will be wrong for some cases so we'd rather keep it as-is to avoid the breaking change. You can unregister and re-register these in the config you need if the default doesn't work for you: https://github.com/gitpod-io/gitpod/blob/58e1f558c90aa819b6061ce170ad7b0320183dec/components/ws-daemon/pkg/daemon/daemon.go#L63
/hold

Reply:
@dongjiang1989 Was chatting with Alvaro.

In general we don't want to finetune things like this too much as it's hard to find something that works for everyone.

But in this specific case we're good. It did some research and it turns out we can simply enable all Go metrics as there are not that many. SO it's not too much overhead. And having all Go metrics should of course work for everyone

Please change this to:
```go
collectors.NewGoCollector(collectors.WithGoCollectorRuntimeMetrics(collectors.MetricsAll))
```

(btw component-base, i.e. kube-apiserver, ... are doing the same)

/hold cancel

Reply:
Thanks @alvaroaleman and @sbueringer 
Changed it